client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -347,6 +348,10 @@ const TransferPage = ( props: TransferPageProps ): JSX.Element => {
 			return <AftermarketAutcionNotice domainName={ selectedDomainName } />;
 		}
 
+		if ( domain.expired && ! domain.isRenewable ) {
```

Comment:
Can we replace this condition with `domain.isRedeemable` ?

Reply:
@delputnam I think that if we use only `domain.isRedeemable` we are going to miss this case (domain expired and after redeemable period). 

| expired | isRenewable | isRedeemable | can transfer |
|---------|-------------|--------------|--------------|
| true    | false       | false        | false        |

I'm not sure if the domain is immediately removed from calypso: if it is, then is ok tu use only `domain.isRedeemable` 👍

Reply:
Updating conditions to show the non-transferrable domain notice to use only `domain.isRedeemable`